.github/workflows/pr-opened-help-message.yml

```diff
@@ -0,0 +1,31 @@
+name: Pull Request Opened Help Message
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  pr-opened-help-message:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Comment on PR
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `Hello 👋! Thanks for opening a pull request. 
+              
+              Please make sure to follow the [contribution guidelines](https://opentelemetry.io/docs/contributing/). 
+                
+              We will run some checks on your PR. If they fail, take a look in to our [guide on pr-checks](https://opentelemetry.io/docs/contributing/pr-checks/). 
+              Most issues can be resolved by running \`npm run fix:all\` locally and pushing the changes. 
+              But please note that it is not your responsibility to fix all the issues. We will help you with that. 
```

Comment:
```suggestion
              We maintainers can sometimes help you resolve issues with your PR, but fixing them is up to you.
```

Reply:
I phrased it that way because depending on the issue I rather fix it myself vs leaving it to an (unexperienced) contributor. It can be very frustrating for them to get the checks all satisfied. They create a lot of notifications for us and also a lot of CI check runs that clog the github workers.

A different wording would be:

Do not worry about the status of the CI checks after submitting your PR. We will provide you with guidance on how to address them.